src/main/menu_main.cpp

```diff
@@ -331,21 +368,23 @@ bool mainMenuUpdate()
 
         } // No keyboard/Joystick grabbing active
 
+#ifndef __3DS__
         if(SDL_AtomicGet(&loading))
         {
             if((menuDoPress && MenuCursorCanMove) || MenuMouseClick)
                 PlaySoundMenu(SFX_BlockHit);
             if(MenuCursor != 0)
                 MenuCursor = 0;
         }
+#endif
         // Main Menu
         else if(MenuMode == MENU_MAIN)
```

Comment:
Haha, you disabling the block, and missing up the `else if`, that will break the compilation on 3DS itself :grin:

in such case, you should take `else` into the macro too:
```cpp
#ifndef __3DS__
    if(...)
    {
    }
    else
#endif
    if(...)
    {
    }
```

Reply:
Fixed now, thanks!